aiohttp/http_parser.py

```diff
@@ -936,6 +936,7 @@ class DeflateBuffer:
     def __init__(self, out: StreamReader, encoding: Optional[str]) -> None:
         self.out = out
         self.size = 0
+        self.out.total_compressed_bytes = self.size
```

Comment:
```suggestion
       out.total_compressed_bytes = self.size
```

nit: uneeded self access